tests/mail/tests.py

```diff
@@ -276,6 +276,34 @@ def test_unicode_headers(self):
         self.assertEqual(message['Sender'], '=?utf-8?q?Firstname_S=C3=BCrname?= <sender@example.com>')
         self.assertEqual(message['Comments'], '=?utf-8?q?My_S=C3=BCrname_is_non-ASCII?=')
 
+    @override_settings(
+        EMAIL_USE_LOCALTIME=None,
+        USE_TZ=True,
+        TIME_ZONE='Africa/Algiers')
+    def test_date_header_utc(self):
+        """
+        If settings.EMAIL_USE_LOCALTIME is False, return time in UTC
+        """
+        email = EmailMessage(
```

Comment:
`email = EmailMessage('Subject', 'Content', 'bounce@example.com', ['to@example.com'])`